completions/brctl

```diff
@@ -1,4 +1,19 @@
 # bash completion for brctl                                -*- shell-script -*-
+_brctl_interfaces()
```

Comment:
We have some new naming conventions for functions in effect, please refer to https://github.com/scop/bash-completion/blob/master/doc/api-and-naming.md#naming

Reply:
So this one's going to be `_comp_xfunc_brctl_interfaces`? I mean, there is no other consumer now, but makes perfect sense to have a function that returns a list of bridged interfaces. Then, I belive, a check like `[[ -x brctl ]] || return` needs to be added in case brctl isn't installed.

Reply:
Maybe, but then again we can also do that when there is a known consumer for that function. No strong opinions.

The current implementation of the function already redirects stderr to `/dev/null`, so I don't think a separate existence/executability check is required. We tend to not sprinkle those unless there's a specific reason to.

Also, the brctl to invoke should be passed to the function as a parameter. Random in-tree example:
- https://github.com/scop/bash-completion/blob/8490bf806a86a7960f0b9a14e16421ea7bd05ef5/completions/python#L6
- https://github.com/scop/bash-completion/blob/8490bf806a86a7960f0b9a14e16421ea7bd05ef5/completions/python#L39

Reply:
Is this function intended to be exposed to other completions? If not, the function name should be `_comp_cmd_brctl__interfaces` or `_comp_cmd_brctl__network_interfaces`.

Reply:
OK, I'm fixing it. Multitasking isn't my strong suit today, sorry.

Reply:
Thank you!